PIM/UserInterface/02a_List.md

```diff
@@ -0,0 +1,1419 @@
+
+The *Completeness* tab displays the completeness of the product for the different languages and channels. A single box is displayed for each active channel. The tab is only displayed if the completeness of the product has been calculated and is displayed in the header section of the *Edit product* view. Consequently, the tab is never displayed in the *Create product* view.
+
+- Channel name   
+  This box displays the percentage for the product completeness of the selected product regarding the assigned channel.
+  - Language   
+    The box displays the percentage for the product completeness of the selected product regarding the assigned channel and language. A single percentage is displayed for each available language for one channel.
+
+  - ![Info](../../Assets/Icons/Info.png "[Info]") (Info)   
+    Hover over this icon to display a box that is listing the missing attribute values. This icon is only displayed when at least one required attribute value for the product completeness is missing.
+
+
+## Edit product &ndash; Catalogs
+*PIM > Products > Select product > Tab Catalogs*
+
+![Catalogs](../../Assets/Screenshots/PIM/Products/List/Catalogs/CatalogsEdit.png "[Catalogs]")
+
+For each catalog a single box to assign the new product to a category is displayed. You can assign a product to several categories at the same time.
+
+**Catalog Name**
+
+- [EDIT ATTRIBUTE *CATALOG NAME*]   
+  Click this button to assign the product to a category or a sub-category in the selected catalog. The categories window is displayed.
+
+The box displays all selected categories and their level. All fields are read-only. If no category is selected, the notice *No nodes selected* is displayed in the box.
+
+- *Level 1*    
+  Level 1 category name.
+
+- *Level 2*   
+  Level 2 category name. The field is only displayed, if the product is assigned to a category of level 2 or lower.
+
+- *Level x*   
+  Level x category name. The field is only displayed, if the product is assigned to a category of level x or lower. The *x* indicates the level number of the assigned category.
+
+- ![Delete](../../Assets/Icons/Trash01.png "[Delete]") (Delete)   
+  Click this button to delete the selected category from the box. This button is only displayed, when you hover over a category in the box.
+
+
+### Categories
+*PIM > Products > Select product > Tab Catalogs > Button EDIT ATTRIBUTE*
+
+![Categories](../../Assets/Screenshots/PIM/Products/List/Catalogs/Categories.png "[Categories]")
+
+- [x]     
+  Select a checkbox to assign the product to the selected category. You can select multiple checkboxes.
+
+> [Info] Categories that are containing sub-categories are displayed with the symbol ![Folders](../../Assets/Icons/Folders01.png "[Folders]") (Folders), categories without sub-categories with the symbol ![Folder](../../Assets/Icons/Folder01.png "[Folder]") (Folder).    
+Click the [>] button to the left of a category checkbox to display their sub-categories.
```

Comment:
```suggestion
Click the [>] button to the left of a category checkbox to display their sub-categories.
```
vgl. oben (Create - Catalogs - Categories): Click the [>] button to the left of a category checkbox to display all sub-categories.